lib/types.ts

```diff
@@ -4,7 +4,7 @@ import type { Readable } from 'node:stream'
 export interface StorageDriver {
   upload: (buffer: Buffer, objectName: string) => Promise<void> | void
   download: (objectName: string) => Promise<Readable> | Readable
-  prune: () => Promise<void> | void
+  prune: (objectNames: string[]) => Promise<void> | void
```

Comment:
prune bedeutet ja bissl mehr, eig passiert hier nur nen delete oder?
```suggestion
  delete: (objectNames: string[]) => Promise<void> | void
```